server/common/src/main/java/org/eclipse/lsp/cobol/common/copybook/CopybookName.java

```diff
@@ -29,22 +29,22 @@
 @AllArgsConstructor
 @Builder
 public class CopybookName {
```

Comment:
I am not sure if this class was meant to be exposed to the outside world - e.g. the `extension` and any other field added in the future might end up being sent to the client.

Not to mention, if anybody decides to change the names it will break the RPC.

Reply:
What is your suggestion to avoid this ? I don't think it is a good idea to add DTO's without adding some mapper logic.

Reply:
Define a class with only the members expected by the client, document that it is the purpose of the class, use it in downloadCopybooks declaration and calls.

As a bonus, the CopybookName does not have to be touched at all, making this PR half the size.